scripts/lint-md-links.ts

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+import fs from 'fs';
+import path from 'path';
+import sh from 'shelljs';
+
+function getAllMarkdownFiles(directory: string): string[] {
+  const markdownFiles: string[] = [];
+
+  function traverseDirectory(currentPath: string): void {
+    const files = fs.readdirSync(currentPath);
+
+    files.forEach((file) => {
+      const filePath = path.join(currentPath, file);
+      const fileStat = fs.statSync(filePath);
+
+      if (fileStat.isDirectory() && file !== 'node_modules') {
+        traverseDirectory(filePath);
+      } else if (fileStat.isFile() && path.extname(file) === '.md' && file !== 'CHANGELOG.md') {
```

Comment:
I am ignoring our changelogs.

Reply:
For listing all markdown files, we could use the `glob` package:

https://github.com/FuelLabs/fuels-ts/blob/b71ad9fd05f9fa478d66a18739be1a8b7956a4d4/packages/abi-typegen/src/runTypegen.test.ts#L75

It seems the API changed a bit in the last version. 🤔
 - https://github.com/isaacs/node-glob